lib/ask/runtime/panel_survey.ex

```diff
@@ -11,6 +11,11 @@ defmodule Ask.Runtime.PanelSurvey do
       survey
       |> Repo.preload([:project])
 
+    schedule =
+      survey.schedule
+      |> Schedule.remove_start_date()
+      |> Schedule.remove_end_date()
```

Comment:
Let's try to avoid imperative names for functions, since they don't mutate a thing. This two could be `Schedule.without_start_date()`, for example.

Reply:
I see what you mean, but I think this isn't the standard way of naming functions in Elixir.

Here are a couple of examples (the standard library is plenty of them):

 [Map.replace/3](https://hexdocs.pm/elixir/Map.html#replace/3)
 ```
Map.replace(%{a: 1, b: 2}, :a, 3)
%{a: 3, b: 2}
```

[Map.delete/2](https://hexdocs.pm/elixir/Map.html#delete/2)
```
Map.delete(%{a: 1, b: 2}, :a)
%{b: 2}
```